lib/src/SearchBar.style.js

```diff
@@ -35,7 +35,15 @@ export function textInputStyle(fontSize, fontColor) {
   };
 }
 
-export function _shadowStyle(shadowColor) {
+export function _shadowStyle(shadowColor, shadowStyle) {
+  if (shadowStyle) {
```

Comment:
Actually we do not need that at the style.js part. We can completely remove it and make it from SearchBar.js part. Check it out the comment on shadowStyle part please :)